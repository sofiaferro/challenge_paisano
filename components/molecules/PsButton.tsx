import React from 'react';

const PsButton = () => {
  return <div>PsButton</div>;
};

export default PsButton;
