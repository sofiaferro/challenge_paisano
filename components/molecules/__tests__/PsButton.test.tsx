import React from 'react';
import renderer from 'react-test-renderer';

import { describe, expect, it } from '@jest/globals';

import PsButton from '../PsButton';
describe('PsButton', () => {
  it('renders the component', () => {
    const mockCallback = jest.fn();
    const tree = renderer.create(
      <PsButton title={'test button'} onClick={mockCallback} />
    );
    expect(tree).toMatchSnapshot();
  });
});
