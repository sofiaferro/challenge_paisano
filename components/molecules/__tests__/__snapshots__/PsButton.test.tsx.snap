// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PsButton renders the component 1`] = `
<div
  className="sc-21b4ebd5-1 gPuGRa"
  style={{}}
>
  <div
    className="sc-21b4ebd5-0 lmlyrg"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={{}}
  >
    <p>
      test button
    </p>
  </div>
</div>
`;
